realm/realm-jni/src/io_realm_internal_table.cpp

```
@@ -823,6 +823,21 @@ JNIEXPORT void JNICALL Java_io_realm_internal_Table_nativeSetByteArray(
     } CATCH_STD()
 }
 
+JNIEXPORT void JNICALL Java_io_realm_internal_Table_nativeSetNull(
+    JNIEnv* env, jobject, jlong nativeTablePtr, jlong columnIndex, jlong rowIndex)
+{
+    Table* pTable = TBL(nativeTablePtr);
+    if (!TBL_AND_COL_INDEX_VALID(env, pTable, columnIndex))
+        return;
```

Now this just fails silently. Is that a good idea?